packages/components/inputs/creatable-select-input/src/creatable-select-input.tsx

```diff
@@ -223,167 +93,164 @@ CreatableSelectInput.propTypes = {
   /**
    * Aria label (for assistive tech)
    */
-  'aria-label': PropTypes.string,
+  'aria-label': Props['aria-label'];
   /**
    * HTML ID of an element that should be used as the label (for assistive tech)
    */
-  'aria-labelledby': PropTypes.string,
+  'aria-labelledby': Props['aria-labelledby'];
   /**
    * Focus the control when it is mounted
    */
-  isAutofocussed: PropTypes.bool, // original: autoFocus
+  isAutofocussed?: boolean; // original: autoFocus
   /**
    * Remove the currently focused option when the user presses backspace
    */
-  backspaceRemovesValue: PropTypes.bool,
+  backspaceRemovesValue?: Props['backspaceRemovesValue'];
   /**
    * Map of components to overwrite the default ones, see [what components you can override](https://react-select.com/components)
    */
-  components: PropTypes.objectOf(PropTypes.func),
+  components?: Props['components'];
   /**
    * Custom method to filter whether an option should be displayed in the menu
    */
-  filterOption: PropTypes.func,
+  filterOption?: Props['filterOption'];
   // This forwarded as react-select's "inputId"
   /**
    * The id of the search input
    */
-  id: PropTypes.string,
+  id?: Props['inputId'];
   // This is forwarded as react-select's "id"
-  inputValue: PropTypes.string,
+  inputValue?: Props['inputValue'];
   /**
    * The id to set on the SelectContainer component
    */
-  containerId: PropTypes.string,
+  containerId?: Props['id'];
   /**
    * Is the select value clearable
    */
-  isClearable: PropTypes.bool,
+  isClearable?: Props['isClearable'];
   /**
    * Is the select disabled
    */
-  isDisabled: PropTypes.bool,
+  isDisabled?: Props['isDisabled'];
   /**
    * Override the built-in logic to detect whether an option is disabled
    */
-  isOptionDisabled: PropTypes.func,
+  isOptionDisabled?: Props['isOptionDisabled'];
   /**
    * Support multiple selected options
    */
-  isMulti: PropTypes.bool,
+  isMulti?: Props['isMulti'];
   /**
    * Whether to enable search functionality
    */
-  isSearchable: PropTypes.bool,
+  isSearchable?: Props['isSearchable'];
   /**
    * Maximum height of the menu before scrolling
    */
-  maxMenuHeight: PropTypes.number,
+  maxMenuHeight?: Props['maxMenuHeight'];
   /**
    * Dom element to portal the select menu to
    */
-  menuPortalTarget: PropTypes.instanceOf(SafeHTMLElement),
+  menuPortalTarget?: Props['menuPortalTarget'];
   /**
    * z-index value for the menu portal
    */
-  menuPortalZIndex: PropTypes.number.isRequired,
+  menuPortalZIndex?: number;
```

Comment:
I think you missed this: prop type was initially required but it's now optional here.